tests/parm/v16_c96_rrtmgp.nml.IN

```diff
@@ -199,7 +199,7 @@ deflate_level=1
   sw_file_gas    = 'rrtmgp-data-sw-g224-2018-12-04.nc'
   sw_file_clouds = 'rrtmgp-cloud-optics-coeffs-sw.nc'
   doG_cldoptics = .true.
-  rrtmgp_ngauss_ang = 3
+  rrtmgp_ngauss_ang = 1
```

Comment:
@dustinswales Do we want to include this change in this PR? If so should we not use the same setting in every namelist, e.g. the new one you've provided for RRFS?

Reply:
@RobertPincus
Good catch. This did not belong.